### ISSUE TITLE
Token#getClaims() returns unexpected values

### ISSUE THREAD

**Issue by schechter** [lcobucci/jwt#593]
We've just had to pin to 3.3.3 as 3.4.1 is not behaving as expected.

`getClaims` no longer returns the expected value (perhaps only when the value is multiple lines?)

**Comment by lcobucci** [comment]
That's extremely unexpected, would be able to reproduce it in a test case?

**Comment by schechter** [comment]
I'll see if I can get something together, in the mean time here's some more details.
Earlier today we started getting errors in one of our staging environments.
Our composer files had `"lcobucci/jwt": "^3.2",`
The images running in production were running `"source": "https://github.com/lcobucci/jwt/tree/3.3.3"` while the environments with this errors were running `"source": "https://github.com/lcobucci/jwt/tree/3.4.1"`

We updated the composer file to pin the version to 3.3.3 and the new images resolved the issue.
I'll have to create a jwt that has a similar structure to create a test case for you.
